app/components/Edit/EditSections.js

```diff
@@ -330,21 +337,23 @@ class EditSections extends React.Component {
   }
 
   handleDeactivation(type, id) {
-    let path = null;
-    if (type === 'resource') {
-      path = `/api/resources/${id}`;
-    } else if (type === 'service') {
-      path = `/api/services/${id}`;
-    }
-    dataService.APIDelete(path, { change_request: { status: "2" } })
-    .then(() => {
-      alert('Successfully deactivated! \n \nIf this was a mistake, please let someone from the ShelterTech team know.')
-      if(type === 'resource') {
-        this.props.router.push({ pathname: "/" });
-      } else {
-        window.location.reload();
+    if (confirm('Are you sure you want to deactive this resource?') === true) {
+      let path = null;
+      if (type === 'resource') {
+        path = `/api/resources/${id}`;
+      } else if (type === 'service') {
+        path = `/api/services/${id}`;
       }
-    });
+      dataService.APIDelete(path, { change_request: { status: "2" } })
+      .then(() => {
+        alert('successfully deactivated! \n \nif this was a mistake, please let someone from the sheltertech team know.')
```

Comment:
Can you use proper capitalization here? Since this is user-facing, we should make it look professional.

Reply:
Definitely! Good catch!